lib/lunchbot_web/templates/admin/select_menu/view.html.heex

```diff
@@ -8,7 +8,7 @@
     <% end %>
 
     <%= label f, :office %>
-    <%= select f, :offices, ["Providence": 1, "Boulder": 2], required: true %>
+    <%= select f, :offices, @offices, required: true %>
```

Comment:
Was this perhaps the actual problem? What offices are seeded with a fresh database? What happens if you update the static list of options to reflect that?